.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
+<!--  
+Thanks for submitting a pull request!
 
-1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
-https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
-3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
-4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-5. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+If this is your first time submitting a PR, please familiarise yourself with:
+
+  Contributor Guidelines 
+  https://git.k8s.io/community/contributors/guide#your-first-contribution
+
+  Developer Guide 
+  https://git.k8s.io/community/contributors/devel/development.md#development-guide
+
+The following are additional useful links for PRs:
+
+  Best Practices for Faster Reviews 
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+
+  Marking Unfinished PRs for Review or as a Work in Progress (WIP)
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
```

Comment:
A PR can touch areas that target multiple SIGs and we already have this information with the `sig/` labels, so asking to mention the SIGs here would lead to multiple sources of truth :grimacing:

Reply:
+1, this is unneeded.

Reply:
@nikhita - this was specifically targeting the primary sig based on the request in [issue #668](https://github.com/kubernetes/sig-release/issues/668). @onyiny-ang - it may be worth discussing this in more detail. Looking at the [release-notes-draft.md](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.16/release-notes-draft.md), I'm not sure I can see the reason for the primary sig collection.

@mrbobbytables - I have been wondering how that aspect of the label magic worked. Thanks 👍

Reply:
Sorry in advance because this will be a bit long winded, but from the release notes standpoint, we were hoping to collect the _singular SIG_ that is primarily responsible for the change the PR brings so that when we are sorting release notes (or ideally programming the release notes tool to collect release notes for a release) we are able to easily identify which SIG the note should be filed under. Having multiple SIG labels helps identify all areas that a PR may impact or contribute changes to and the [release notes website](relnotes.k8s.io) makes this easier for users to digest. 

However, for the release notes draft, the relnotes tool sorts each note that is tagged with multiple SIG labels to buckets created for that note (ie. there will be a SIG API Machinery bucket for PRs with only that SIG label as well as a bucket for all notes labelled with both SIG API Machinery and SIG Cluster Lifecycle)  and this makes things confusing for the relnotes team and contributors trying to find all of the pertinent notes associated with what their SIG has accomplished/changed for a release. For the relnotes team, which may be fairly new contributors, it can be impossible to determine which SIG to sort the PR under without reaching out to every contributor who submitted a PR tagged with multiple labels.

As we're still generating the human curated notes, it would greatly help the relnotes team and contributors who have to look through the release notes draft if we were able to identify the SIG that the release note should be filed under automatically.  This was what I had in mind when I was requesting the "primary-sig"  field to be added. BUT I'm certainly open to better ideas!

Reply:
It seems like this is a non-issue for the [release notes website](https://relnotes.k8s.io/). It is just the markdown release notes draft where this is an issue. And given how many of the current release notes fall into multiple sigs, I can see how this can create a lot of manual work.